### ISSUE TITLE
get-BCArtifactUrl -select daily

### ISSUE THREAD

**Issue by NAVFreak** [microsoft/navcontainerhelper#2915]
**Describe the issue**
It would have been nice if you could run

`Get-BCArtifactUrl -type "Onprem"   -country "se"   -select "daily"`

However, this isn't the real issue.
I'm missing documentation for daily and weekly.
![image](https://user-images.githubusercontent.com/34970096/219363134-2bf98d3a-9a1a-46f1-9626-e30ea16364a9.png)

First I thought daily and weekly were special ALOps parameters. But then it took me some troubleshooting to realise it is part of the bccontainerhelper and that you can't combine daily with type Onprem. 

Currently this happens when you do it:
```
>Get-BCArtifactUrl -type "Onprem"   -country "se"   -select "daily"
_Get-BCArtifactUrl Telemetry Correlation Id: cc0cdf03-337f-477d-9e3d-7f6dd96fedfa
You cannot specify type=OnPrem, version, before or after  when selecting Daily or Weekly build
At C:\Users\margus\Documents\WindowsPowerShell\Modules\BcContainerHelper\4.0.10\Artifacts\Get-BCArtifactUrl.ps1:74 char:13
+             throw 'You cannot specify type=OnPrem, version, before or ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (You cannot spec...or Weekly build:String) [], RuntimeException
    + FullyQualifiedErrorId : You cannot specify type=OnPrem, version, before or after  when selecting Daily or Weekly build_
```

**To summarise:** Add documentation of daily and weekly and a "nice to have" feature to use combiation -type "Onprem"  -select "daily"` ;)

**Comment by freddydk** [comment]
What would you expect when using daily or weekly with -type onprem?
The reason for daily and weekly is that we release several builds every day - and daily+weekly ensures you have the same artifacturl throughout the day or week.
on-premises builds are released monthly and as such, doesn't have the same issue as sandbox builds.

**Comment by freddydk** [comment]
Will update the docs and see whether I an enable them to work on on-premises artifacts and basically return the same artifacts for a day or a week - ignoring if a new release was released...

**Comment by NAVFreak** [comment]
Oh, didn't know that onprem was released monthly. 
But that would indeed be nice if onprem daily/weekly took the montly.

**Comment by freddydk** [comment]
Shipped in 5.0.0
